--- a/DockerContainers.php
+++ b/DockerContainers.php
@@ -131,7 +131,7 @@
   $id = $image['Id'];
   $menu[] = sprintf("addDockerImageContext('%s','%s');", $id, implode(',',$image['Tags']));
   echo "<tr class='advanced'><td style='width:220px;padding:8px'>";
-  echo "<span class='outer apps'><span id='$id' class='hand'><img src='/webGui/images/disk.png' class='img'></span><span class='inner'>(orphan image)<br><i class='fa fa-square stopped grey-text'></i><span class='state'>stopped</span></span></span>";
+  echo "<span class='outer apps'><span id='$id' class='hand'><img src='/plugins/dynamix.docker.manager/images/image.png' class='img'></span><span class='inner'>(orphan image)<br><i class='fa fa-square stopped grey-text'></i><span class='state'>stopped</span></span></span>";
   echo "</td><td colspan='5'>Image ID: $id<br>";
   echo implode(', ',array_map('htmlspecialchars',$image['Tags']));
   echo "</td><td>Created ".htmlspecialchars($image['Created'])."</td></tr>";
